src/assets/scss/_recordedit.scss

```diff
@@ -104,28 +104,151 @@
 
         &.highlighted-row {
           background-color: map-get(variables.$color-map, 'recordedit-highlighted-row');
+          // We add this so that the select-all-row spans across the entire view
+          max-width: none;
+
+          .chaise-btn-toolbar {
+            pointer-events: none;
+          }
         }
 
         .inputs-row {
           display: flex;
           flex-direction: row;
+
+          // adding an overlay on top of form to add the click event
+          .form-overlay {
+            height: 100%;
+            width: 100%;
+            cursor: pointer;
+          }
         }
 
         .select-all-row {
           width: 100%;
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          gap: 5px;
+          // Making it sticky so that it is fixed in its position and rest of the form conatiner scrolls horizontally
+          position: sticky;
+          top: 0;
+          left: 0;
+          z-index: 2;
+
+          // Adding a div to align the content centre always
+          .centre-align {
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            gap: 5px;
+            max-width: 100%;
+          }
 
-          .select-all-text, .select-all-input, .select-all-buttons {
-            float: left;
+          .centre-align-textarea {
+            width: auto;
```

Comment:
This is not a child class for centre-align, it is a separate class for textarea

Reply:
I know. `.centre-align-textarea` is a more specific class that you're adding to only some `.centre-align` elements. That's why I wrote it as `&.centre-align-textarea`.

In scss just because you're writing a rule inside another one doesn't mean that it will always go another level deep. It just means start from there. `&` in here refers to `.centre-align` elements and by appending `.centre-align-textarea` to it, we're basically saying all the `.centre-align.centre-align-textarea` elements.

If I you were to just say `.centre-align-textarea` or `& .centre-align-textarea` then it would be different and not what we want. 

Also let's use `center` instead of `centre`.